doc/content/getting-started/aws/ecs/interop/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Interoperability"
+description: "LoRaWAN Backend Interfaces"
+weight: 3
+---
+
+{{% tts %}} offers support for LoRaWAN Backend Interfaces. Interoperability can be turned on on JoinServer or IdentityServer, but not both at once, as they use the same port.
+
+# JoinServer interoperability
+
+## Architecture
+
+Support for LoRaWAN Backend Interfaces requires certain extensions to the typical AWS ECS deployment architecture. First, we need to include CryptoServer in the deployment. This component is responsible for cryptographic operations in the context of Microchip secure element. Since it needs access to highly sensitive data, {{% tts %}} assumes that it's not deployed together with the rest of the stack, but in a separate account. In this way regular ops team can have access to the most part of the stack, while CryptoServer, together with highly sensitive Microchip secure element encryption keys, would be accessed only by selected members of your organization.
+
+In order to provide this, {{% tts %}} uses VPC peering functionality of AWS.
```

Comment:
Remove; this is not relevant for public documentation. Also, how things are operated in TTSC(E) are not documented publicly.